android/build.gradle

```diff
@@ -49,7 +49,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.github.doublesymmetry:kotlinaudio:1.2.4'
+    implementation 'com.github.DoubleSymmetry:KotlinAudio:544e3ae497b7f7e657e60464c5a639df1f64e90d'
```

Comment:
I don't think you should change the casing of `DoubleSymmetry` I believe it's case-sensitive.